tokenLists/sei.json

```diff
@@ -1032,5 +1032,12 @@
     "token": "sei1j877q3qct6x2ksgnem3dmz73e0q9hxyry6a2twev5z5xxw5s62rs69f00n",
     "icon": "/img/SeiLoot.png",
     "decimals": 6
+  },
+  {
+    "protocol": "F*ck Scammer",
```

Comment:
We generally are very lax on content, but profanity isn't something we like on Astroport UI.